ooniprobe/AppDelegate.mm

```diff
@@ -34,6 +37,10 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     return YES;
 }
 
+- (BOOL)application:(UIApplication *)application continueUserActivity:(NSUserActivity *)userActivity  restorationHandler:(void (^)(NSArray *restorableObjects))restorationHandler{
```

Comment:
From the [Apple docs](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623072-application?preferredLanguage=occ) I read:
> The app calls this method when it receives data associated with a user activity. For example, when the user uses Handoff to transfer an activity from a different device. 

I also read that it has to do with SiriKit. Maybe I've not digged down enough, but it is my understanding that this method returning `YES` is required to continue inside the app the activity initiated clicking in the URL.

Correct?

Reply:
Answered above. We only handle https://run.ooni.io here